README.md

```diff
@@ -65,6 +67,8 @@ The branch name is used by Cloudflare Pages to determine if the deployment is pr
 If you are in a Git workspace, Wrangler will automatically pull the branch information for you. You can override this
 manually by adding the argument `branch: YOUR_BRANCH_NAME`.
 
+When using a branch other than `main` for your production branch, you must set `branch: main` in your workflow to ensure that the deployment is published to the production environment.
```

Comment:
The production branch is defined by the first branch you push to within your Pages project or it's `main` if made through the dash.

For most people, this will be whatever their actual production branch is called.

Reply:
Aha, well that makes this pr null then, This should probably be explained a little better - Thought I could also have easily missed this if it is in the docs. Will close the PR.